packaging/CPackOptions.cmake

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019-2019 the openage authors. See copying.md for legal info.
+
+# Keeps all required CPack configuration for the Generators in one place and
+# CPack calls this file for every generator, keep this in mind, if you use
+# variables or functions outside of the conditional branch
+
+
+########################################################################
+# Continuous Integration
+
+# Filename addition for architecture, set from environment variables
+set(CPACK_PACKAGE_ARCHITECTURE "$ENV{TARGET_PLATFORM}")
+
+# Filename addition for nightly-builds, set from environment variables
+if(DEFINED ENV{IS_NIGHTLY})
+    set(PACKAGE_VERSION_STRING "${CPACK_VERSION_FULL_STRING}")
+    set(PACKAGE_NIGHTLY "_NIGHTLY")
+else()
+    set(PACKAGE_VERSION_STRING "v${PROJECT_VERSION}")
+    message(DEBUG "${PACKAGE_VERSION_STRING}")
+    set(PACKAGE_NIGHTLY "")
+endif()
```

Comment:
If it's nightly build, then use long version, if release we can just use the shorter version string